vectornav/src/vn_sensor_msgs.cc

```diff
@@ -180,8 +180,11 @@ class VnSensorMsgs : public rclcpp::Node
       q_ned2enu.setRPY(M_PI, 0.0, M_PI / 2);
       msg.orientation = toMsg(q_ned2enu * q);
 
-      msg.angular_velocity = msg_in->angularrate;
-      msg.linear_acceleration = msg_in->accel;
+      // NED to ENU conversion
+      // swap x and y and negate z
+      msg.angular_velocity.x = msg_in->angularrate.y;
+      msg.angular_velocity.y = msg_in->angularrate.x;
+      msg.angular_velocity.z = -msg_in->angularrate.z;
 
       fill_covariance_from_param("orientation_covariance", msg.orientation_covariance);
       fill_covariance_from_param("angular_velocity_covariance", msg.angular_velocity_covariance);
```

Comment:
Was wondering if that needs swapping too. Just to be sure, from what I can see, we use the default covariances and they are represented as a [scalar matrix](https://github.com/botsandus/vectornav/blob/AUTO-538-fix-imu-coordiante-system)/vectornav/config/vectornav.yaml) so there is nothing to change, right? 

It's probably worth mentioning this upstream so that people getting this fix do not work with covariances (although they might have some buggy code anyway).

Reply:
The covariance is read in order from the parameter so we don't need to make in changes here and the default parameters are the same for all axes. 

Will definitely feed this through upstream, I might make the NED->ENU conversion an optional parameter thing because some savages may still want it in ENU.